featurebyte/service/target_table.py

```diff
@@ -0,0 +1,81 @@
+"""
+TargetTableService class
+"""
+from __future__ import annotations
+
+from typing import Optional
+
+from pathlib import Path
+
+import pandas as pd
+from bson import ObjectId
+
+from featurebyte.enum import MaterializedTableNamePrefix
+from featurebyte.models.base import FeatureByteBaseDocumentModel
+from featurebyte.models.target_table import TargetTableModel
+from featurebyte.schema.target_table import TargetTableCreate
+from featurebyte.schema.worker.task.target_table import TargetTableTaskPayload
+from featurebyte.service.materialized_table import BaseMaterializedTableService
+from featurebyte.storage import Storage
+
+
+class TargetTableService(BaseMaterializedTableService[TargetTableModel, TargetTableModel]):
```

Comment:
This service also seems almost identical to HistoricalTableService. Is there there a plan to refactor and consolidate?